06-classification_continued.Rmd

```diff
@@ -99,7 +102,7 @@ you want to trade off between training an accurate model (by using a larger
 training data set) and getting an accurate evaluation of its performance (by
 using a larger test data set).  Here, we will use 75% of the data for training,
 and 25% for testing.  To do this we will use the `initial_split` function, 
-specifying that `prop = 0.75` and the target variable is `Class`:
+specifying that `prop = 0.75` and the response variable is `Class`:
 ```{r 06-initial-split}
 set.seed(1)
 cancer_split <- initial_split(cancer, prop = 0.75, strata = Class)
```

Comment:
A list here for the steps below (again, GitHub won't let me put inline comments on things that haven't been changed recently).

In 2. Pre-process the data we write that "An additional consideration we need to take when doing this is that we should create the standardization preprocessor using only the training data. This ensures that our test data does not influence any aspect of our model training." but will students remember how data is standaridized? We might need to remind them how this is done, and how including the test data will change/influence this?

Reply:
Anytime below we write "K-nearest neighbour", let's change it to "KNN".

Reply:
In 3 should we narrate and remind them that to setup the classifier as before, we need a model spec, and then we combine our recipe from the preprocessing with our model spec in a workflow, which we then pass to `fit` to get our trained classifier object?

Reply:
At the bottom of 3, in the note we write: 

Note that in your own code, you only have to set the seed once at the beginning of your analysis.

That's not entirely true. In a script yes, in a notebook you did restart + run all yes, but not in a notebook where they might run cells independently multiple times... I think in our assignments we put a `set.seed` before each random function to guard against that...

Reply:
In the first paragraph from 4, we write "...while the `.pred_class` contains the predicted diagnoses from the model." but I think we should write "...while the `.pred_class` contains the predicted diagnoses from the classifier."

Reply:
After we call and view the output of `metrics`, I think we should explain that the first row, where it says accuracy, the value for accuracy is in the `.estimate` column. Maybe we should comment on why it is labelled as estimate... For example, why is that a reasonable name (kind of nice foreshadowing to CV that is coming next I think!). Also, we could mention that `kap` is short for Cohen's kappa - another way you can measure classifier performance that they might learn about in more advanced courses.  Also, what does `binary` mean?

Reply:
rather than tell them about things that we won't be doing anything with, i've just said to focus on the accuracy and ignore the rest (beyond scope).